test/hotspot/jtreg/runtime/cds/appcds/applications/JavacBench.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Since this is a new file, should the copyright year be only 2024?
(same for other files)